src/components/Onfido/BaseOnfidoWeb.tsx

```diff
@@ -47,7 +52,6 @@ function initializeOnfido({sdkToken, onSuccess, onError, onUserExit, preferredLo
             colorBorderLinkUnderline: themeColors.link,
             colorBackgroundLinkHover: themeColors.link,
             colorBackgroundLinkActive: themeColors.link,
-            authAccentColor: themeColors.link,
```

Comment:
This param wasn't found by TS, I haven't found it in lib/docs as well

Reply:
ok, but we should make sure isn't being used in Onfido SDK, because it can be an issue with TS typings but not the code itself.

Reply:
Onfido SDK source code is not is not publicly available to check this param there, but I can see the last version this params [was mentioned](https://github.com/onfido/onfido-sdk-ui/blob/10.1.0/UI_CUSTOMIZATION.md) in Onfido Readme is 10.1.0, after that it wasn't there anymore.